### PR TITLE
[WIP]Adding odo interactive mode testing

### LINKED ISSUE #2194: odo Interactive mode test script implementation
Add integration test for odo interactive mode usecase

### PR DESCRIPTION
**Pull request by amitkrout**
## What is the purpose of this change? What does it change?

Interactive mode testing of odo commands

## Was the change discussed in an issue?
fixes #2194

## How to test changes?

// TODO

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -62,3 +62,5 @@ import:
   repo: https://github.com/openshift/gssapi.git
 - package: github.com/kylelemons/godebug
   version: fa7b53cdfc9105c70f134574002f406232921437
+- package: github.com/AlecAivazis/survey
```
Comment (kadel) [inline glide.yaml]: This is duplicated entry for `survay` We are already using `survey` package `v1` `"gopkg.in/AlecAivazis/survey.v1"` We should use just one version!

**Comment by cdrage** [comment]
Make sure you make separate commits between the vendor and actual code being created for testing interactive mode! Just tried to review but couldn't since it was all vendor code.

**Comment by amitkrout** [comment]
> Make sure you make separate commits between the vendor and actual code being created for testing interactive mode! Just tried to review but couldn't since it was all vendor code.

The code commit has been messed up, will clean up and push atleast a minimal running running test for review.

**Comment by amitkrout** [comment]
https://github.com/openshift/odo/issues/2194#issuecomment-550325068
will open a new pr when #2194 comes to live again
/close

**Pull request closed without merging**